mtags/src/main/scala-3/scala/meta/internal/pc/completions/CompletionValue.scala

```diff
@@ -80,8 +80,10 @@ object CompletionValue:
       label: String,
       symbol: Symbol,
       override val snippetSuffix: CompletionSuffix,
+      maybeApplyOwner: Option[Symbol] = None,
   ) extends Symbolic:
     override def isFromWorkspace: Boolean = true
+    val applyOwner: Symbol = maybeApplyOwner.getOrElse(symbol)
```

Comment:
Maybe we could have `importSymbol` with a default of `symbol` and put it into the main Symbolic trait?